src/winforms/toga_winforms/widgets/datepicker.py

```diff
@@ -0,0 +1,19 @@
+from toga_winforms.libs import WinForms
+from travertino.size import at_least
+
+from .base import Widget
+
+
+class DatePicker(Widget):
+    def create(self):
+        self.native = WinForms.DateTimePicker()
```

Comment:
Yep thats right, it can do date, datetime and time. So the weird thing is you can explicitly set it to only do time but you cannot set it to only do date. The screenshot shows that it only gives you the option to select the date but it will return a datetime object and use the time you started the program as the time regardless of what date you select. 

I think for this one, i just want it to be a date selector only, but since there is no way to explicitly select date only with the windows widget,  I will change the get_value function to return only a date object. I also plan on implementing a DateTimePicker, and a TimePicker after this one.